### PR TITLE
UPDATE Qt-Advanced-Docking-System to v3.7.2

### PR DESCRIPTION
**Pull request by vincent-nv**
**Describe the pull request**

- #### What does your PR fix?  
UPDATE Qt-Advanced-Docking-System to v3.7.2

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
x64-windows, No

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  IDN

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
vcpkg x-add-version done

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

**Comment by JackBoosY** [comment]
Thanks for your contributor.
Please sign CLA first.

**Comment by vincent-nv** [comment]
CLA signed

File: ports/qt-advanced-docking-system/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "qt-advanced-docking-system",
-  "version-string": "3.6.3",
-  "port-version": 2,
+  "version-string": "3.7.2",
```
Comment (JackBoosY) [inline ports/qt-advanced-docking-system/vcpkg.json:3]:
```suggestion
  "version": "3.7.2",
```
Same with you previous PR 😄

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: ports/qt-advanced-docking-system/vcpkg.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "qt-advanced-docking-system",
-  "version-string": "3.6.3",
-  "port-version": 2,
+  "version": "3.7.2",
+  "port-version": 0,
   "description": "Create customizable layouts using an advanced window docking system similar to what is found in many popular IDEs such as Visual Studio",
   "homepage": "https://github.com/githubuser0xFFFF/Qt-Advanced-Docking-System",
   "dependencies": [
```
Comment (JackBoosY) [inline ports/qt-advanced-docking-system/vcpkg.json:7]: Also, you need add the following code:
```json
    {
      "name": "vcpkg-cmake",
      "host": true
    },
    {
      "name": "vcpkg-cmake-config",
      "host": true
    }
```

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: ports/qt-advanced-docking-system/portfile.cmake [hunk 3]
```diff
@@ -1,27 +1,26 @@
...
-vcpkg_install_cmake()
+vcpkg_cmake_install()
 
 file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
 file(INSTALL ${SOURCE_PATH}/gnu-lgpl-v2.1.md DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT})
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/license)
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/share)
-vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/qtadvanceddocking TARGET_PATH share/qtadvanceddocking)
+vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/qtadvanceddocking TARGET_PATH share/qtadvanceddocking)
```
Comment (JackBoosY) [inline ports/qt-advanced-docking-system/portfile.cmake:26]:
```suggestion
vcpkg_cmake_config_fixup(PACKAGE_NAME qtadvanceddocking CONFIG_PATH lib/cmake/qtadvanceddocking)
```

**Review by JackBoosY — CHANGES_REQUESTED** [review]

**Review by JackBoosY — APPROVED** [review]

**Comment by JackBoosY** [comment]
LGTM.

**Comment by BillyONeal** [comment]
Thanks for the update!

**Pull request merged**